features/step_definitions/schools/onboarding_steps.rb

```diff
@@ -40,8 +40,8 @@
 Given "I have completed the Candidate Requirements selection step" do
   steps %(
     Given I am on the 'candidate requirements selection' page
-    And I check "They must apply or have applied to your or a partner school's teacher training course"
-    And I check "They must live locally"
+    And I check "Candidates must apply to our (or a partner school's) teacher training course"
```

Comment:
Yep, that's fine. As mentioned on the Google doc, ideally we'd render the content slightly differently on the candidate side: 

'You must: 
- apply to our teacher training course...
- etc'